source/features/navigation/store.ts

```diff
@@ -61,6 +61,20 @@ export class NavigationStore extends Store {
     );
   }
 
+  private updateGAEvents() {
+    try {
+      const windowObject: Window | any = window;
+      windowObject.gtag('config', environment.GA_TRACKING_ID, {
+        hitType: 'pageview',
+        page_location: location.pathname,
+        title: document.title,
+      });
+    } catch (error) {
+      // silences the error in dev mode
```

Comment:
> if gtag fails to load

This seems like a condition we want to do something about, and we shouldn't be here if there's no tracking ID set. I'm surprised this is even accepted by eslint

Reply:
Fixed